ega-data-api-filedatabase/src/main/java/eu/elixir/ega/ebi/downloader/service/internal/LocalEGAFileServiceImpl.java

```diff
@@ -103,6 +104,11 @@ public void setRestTemplate(RestTemplate restTemplate) {
         this.restTemplate = restTemplate;
     }
 
+    @Override
+    public Iterable<FileKey> getKeyIdByFileId(String fileID) {
```

Comment:
There could be some confussion between fileid and file stableid because you may think that the fileid is the database id as it seems the case with the keyid in the encryption_key_id

Reply:
The fileId is like ''EGAF00000000014' and the same convention is used everywhere the stableId is not used much in code.